src/cli/cli.cpp

```diff
@@ -2149,6 +2150,7 @@ template <> otError Interpreter::Process<Cmd("child")>(Arg aArgs[])
     linkMode.mDeviceType   = childInfo.mFullThreadDevice;
     linkMode.mNetworkData  = childInfo.mFullThreadDevice;
     OutputLine("Mode: %s", LinkModeToString(linkMode, linkModeString));
+    OutputLine("Csl Synchronized: %d ", childInfo.mIsCslSynced);
```

Comment:
nit: should it be `CSL`?

Reply:
I think 'Csl Synchronized' is better because the field indicates if the child is in CSL synchronization with the parent.

Reply:
I mean should `CSL` be in upper case for all 3 letters?

Reply:
Updated.